src/fibsem_tools/io/fibsem.py

```diff
@@ -43,26 +60,26 @@ def to_native_types(self):
 
 
 class FIBSEMData(np.ndarray):
-    """Subclass of ndarray to attach header data to fibsem data"""
+    """Subclass of ndarray to attach header data to fibsem data. Note: this object is deprecated and will soon be removed."""
 
-    def __new__(cls, input_array, header=None):
+    def __new__(cls, input_array, attrs=None):
         # Input array is an already formed ndarray instance
         # We first cast to be our class type
         obj = np.asarray(input_array).view(cls)
         # add the new attribute to the created instance
-        obj.header = header
+        obj.attrs = attrs
         # Finally, we must return the newly created object:
         return obj
 
     def __array_finalize__(self, obj):
         # see InfoArray.__array_finalize__ for comments
         if obj is None:
             return
-        self.header = getattr(obj, "header", None)
+        self.atrs = getattr(obj, "attrs", None)
```

Comment:
```suggestion
        self.attrs = getattr(obj, "attrs", None)
```

This looks like a typo